src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -135,10 +135,8 @@ public AddressBook load() throws StorageOperationException {
 
         // create empty file if not found
         } catch (FileNotFoundException fnfe) {
-            final AddressBook empty = new AddressBook();
-            save(empty);
-            return empty;
-
+            throw new StorageOperationException("Error finding file: " + path);
```

Comment:
hmmm... this is never encountered, no? 
please recollect the discussion during tutorial on why this happens!
i hope you have understood the problem and the possible solution from the tutorial